src/services/Plumber.ts

```diff
@@ -107,23 +107,34 @@ export default class Plumber {
 
     public draggable(
         uuid: string,
-        start: Function,
-        drag: Function,
-        stop: Function,
-        beforeDrag: Function
+        start: Function = null,
```

Comment:
just a thought for future development from a jsPlumb n00b: we might consider passing this method an options object so it's clearer on call which event handler a passed function is intended for.

Reply:
of course, typing `plumberDraggable` would also help for those of us using vscode

Reply:
Feel like the plumber instance could alternatively just be context-ized

Reply:
Ya perhaps that's a longer-term goal, migrating our services to the new context API?

Reply:
Ya, perhaps :)